### PR TITLE
`authn-jwt` `jwks-uri` with `ca-cert` integration tests

### PR DESCRIPTION
**Pull request by sashaCher**
### Desired Outcome

Test recently added `ca-cert` variable of `authn-jwt`

### Implemented Changes

- Generated static certificate chain
- Added new server to jwks (nginx) service configuration in ci and dev environments `mycompany.local` that's using the certificate chain
- The "old" `jwks` server is already using self signed
- Implement tests according test plan

### Connected Issue/Story

[ONYX-15511](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-15511)

### Definition of Done

- [X] Desired outcome is achieved

#### Changelog

- [ ] The CHANGELOG has been updated, or
- [X] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [X] This PR includes new unit and integration tests to go with the code
  changes, or
- [ ] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [X] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [X] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [X] There are no security aspects to these changes

### CODE REVIEW

File: ci/oauth/jwks/cert-chain/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Regenerating Test Certificates
+To regenerate these certificate:
+
+- Clone https://github.com/conjurdemos/conjur-intro repository.
+- Go to `tools/simple-certificates` directory.
+- Run `./generate_certificates 2 chained`.
+- Copy relevant certificates from `certificates/` directory to here and build full chain bundle
+  ```
+  cp .../certificates/nodes/chained.mycompany.local/chained.mycompany.local.cert.pem ./
```
Comment (tzheleznyak) [inline ci/oauth/jwks/cert-chain/README.md:9]: Can this path moved to a variable?

File: cucumber/authenticators_jwt/features/authn_jwt_ca_cert.feature [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    """
+    And I successfully set authn-jwt "jwks-uri" variable to value "https://jwks/ca-cert.json"
+#    And I fetch root certificate from "https://jwks" endpoint as "self"
+#    And I successfully set authn-jwt "ca-cert" variable value to the "self" certificate
+    When I GET "/authn-jwt/raw/cucumber/status"
+    Then the HTTP response status code is 200
+    And the HTTP response content type is "application/json"
+    And the authenticator status check succeeds
+
+  Scenario Outline: ONYX-15313/6: Self-signed jwks-uri with ca-cert contains bundle includes the valid certificate
```
Comment (tzheleznyak) [inline cucumber/authenticators_jwt/features/authn_jwt_ca_cert.feature:40]: why ONYX-15313/6

**Review by tzheleznyak — DISMISSED** [review]
LGTM

**Review by eladkug — APPROVED** [review]
Tests approved

**Pull request merged**